src/components/Layout/ScrollableParent.tsx

```diff
@@ -3,21 +3,24 @@ import { Css, Properties } from "src/Css";
 
 interface ScrollableParentContextProps {
   scrollableEl: HTMLElement;
+  paddingX: number;
 }
 
 const ScrollableParentContext = createContext<ScrollableParentContextProps>({
   scrollableEl: document.createElement("div"),
+  paddingX: 0,
 });
 
 // Allow any css to be applied to the ScrollableParent container.
 interface ScrollableParentContextProviderProps {
   xss?: Properties;
+  paddingX?: number;
```

Comment:
See other comment about a probably-too-cute way to just have this come from `xss`, but if still keeping this as a separate prop (which is fine/a lot simpler), wdyt about `px` to double down on our truss/tachyons naming?